content/custom-commands/_index.md

```diff
@@ -0,0 +1,9 @@
++++
+archetype = "chapter"
+title = "Custom Commands"
+weight = 3
++++
+
+Custom Commands (CCs) allow you to develop your own commands and behaviours for your server. A command may be as simple as an autoresponder or employ a more complex system. A variety of triggers are available for custom commands and the custom template script provides a wide array of templates and functions for coding advanced commands.
```

Comment:
Remember to hard-break at 120 characters. Also, as much as it pains me,
too, we're using American English in this documentation.